frontend/src/components/cronjob/Details.tsx

```diff
@@ -31,11 +60,74 @@ export default function CronJobDetails() {
 
   const ownedJobs = filterOwnedJobs(jobs);
 
+  function handleCron(suspend: boolean) {
+    const suspendingOrStarting = suspend ? 'Suspend' : 'Resum';
+    cronJob.spec.suspend = suspend;
+    // when we apply the cronjob, we need to remove the resourceVersion because a new resourceVersion is generated
+    // when the cronjob is applied
+    delete cronJob.metadata.resourceVersion;
```

Comment:
I wonder if we should just remove the resource version in the `apply` function itself (so it works for all json).

Reply:
updated